equinox-shipment/Domain/FinalizationTransaction.fs

```diff
@@ -0,0 +1,90 @@
+module FinalizationTransaction
+
+module Events =
+
+    let [<Literal>] CategoryId = "FinalizationTransaction"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | FinalizationRequested of containerId: string * shipmentIds: string[]
+
+        | AssignmentCompleted   of containerId: string * shipmentIds: string[]
+        | FinalizationCompleted of containerId: string
+
+        | RevertRequested       of containerId: string * shipmentIds: string[]
+        | FinalizationFailed    of containerId: string
```

Comment:
not sure I like having FinalizationFailed + FinalizationFailed in the events
can just do a Completed event with no body
the state can track the success/fail

Reply:
I've updated to only contain a single `Completed` event and moved the tracking to the state. Thanks!